modules/data-loading/pages/kafka-loader-user-guide.adoc

```diff
@@ -151,11 +281,11 @@ GSQL > DROP DATA_SOURCE *
 GSQL > DROP DATA_SOURCE ALL
 ----
 
-==== SHOW DATA_SOURCE
+=== SHOW DATA_SOURCE
 
 The `SHOW DATA_SOURCE` command will display a summary of all existing data_sources for which the user has privilege:
```

Comment:
The `SHOW DATA_SOURCE` command will display a summary of all existing data sources for which the user has privileges: